src/providers/ethereum/trezorWallet.ts

```
@@ -0,0 +1,46 @@
+import { FeeMarketEIP1559Transaction } from '@ethereumjs/tx'
```

ESLint

[error] 'FeeMarketEIP1559Transaction' is defined but never used.

(@typescript-eslint/no-unused-vars)